bot/exts/moderation/infraction/_utils.py

```diff
@@ -94,6 +95,11 @@ async def post_infraction(
 
     current_time = arrow.utcnow()
 
+    if is_in_category(ctx.channel, Categories.modmail):
+        jump_url_text = "Infraction issued in a ModMail channel."
+    else:
+        jump_url_text = f"[Click here]({ctx.message.jump_url})"
```

Comment:
I guess infraction messages would never be deleted, would they ? 
otherwise that'll be a blank jump, but i think it's unlikely for something like that to hapen.

Because what I'm thinking is, if an admin or mod expect to find the infraction message, but they don't, it could potentially be confusing.

So storing message id then fetching it would give more context, but again, i think it's almost impossible to happen

Reply:
@vivekashok1221 After some testing, and when I tried deleting the message, the jump URL will take you to the latest message sent in that channel, which is confusing.

Again, this is a very unlikely case to happen so it might not even be worth dealing with

Reply:
Back when we discussed this feature in our internal mod channel, we agreed that it was okay to leave dead links as is because, you're right, it's not something that happens often.

Reply:
Well, in that case, I'll resolve this then.
Thanks !